cirq/__init__.py

```diff
@@ -168,13 +168,24 @@
     moment_by_moment_schedule,
 )
 
+from cirq.sim import (
+    check_state,
+    decode_initial_state,
```

Comment:
This method name is ambiguous out of context. Maybe `initial_state_to_wavefunction` or `state_to_wavefunction` or just `wavefunction(state)` or `state_vector(index=x, vector=y, dtype=z)`?